mkosi/config.py

```diff
@@ -1007,11 +1007,30 @@ class MkosiConfigParser:
             dest="kernel_modules_initrd_include",
             section="Content",
             parse=config_make_list_parser(delimiter=","),
+            default=[
```

Comment:
I'm not sure we should exclude stuff here? Isn't this better done in the configuration in the systemd repo?

Reply:
Given that it's trivial to override this by just assigning to the setting, I think its better to exclude some stuff here. Having initrds of slightly more reasonable size seems like a better default than generating one that includes absolutely everything.

Reply:
For generic, default use case this is a surprising behaviour - we cannot possibly know what somebody might need in the initrd - eg: accessibility tools that need audio for example. For our own use case, like the config used for CI jobs here, and for images we build in the systemd repo, we should totally exclude these.

Reply:
Dropped default exclude list for now and bumped ESP size to 512MB instead of 256MB to accomodate